new-packages/ts-project/src/state.ts

```diff
@@ -221,6 +222,16 @@ function registerActionBlocks(
   }
 }
 
+function getDefaultInternalValue(
```

Comment:
This function could use a code comment as to what we mean by `default internal value`.